### PR TITLE
NOISSUE - Fix default values for port and x509 provision

### PR DESCRIPTION
**Pull request by mteodor**
### What does this do?
Make http port values across docs and code consistent.
x509 provision should be false by default.

### Which issue(s) does this PR fix/relate to?
No such issue

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### CODE REVIEW

File: cmd/provision/main.go [hunk 1]
```diff
@@ -48,7 +47,6 @@ const (
 	envTLS              = "MF_PROVISION_ENV_CLIENTS_TLS"
 	envServerCert       = "MF_PROVISION_SERVER_CERT"
 	envServerKey        = "MF_PROVISION_SERVER_KEY"
-	envMQTTURL          = "MF_PROVISION_MQTT_URL"
```
Comment (manuio) [inline cmd/provision/main.go:51]: Should we also remove it from docker-compose and `.env` ?

**Review by nmarcetic — DISMISSED** [review]

**Comment by jonathandreyer** [comment]
@mteodor Thanks for the PR. I have also seen duplicates declaration of environment variables in the README.md of *provision* service (*MF_PROVISION_SERVER_KEY*, *MF_PROVISION_USER* & *MF_PROVISION_PASS*). I also wonder about the two environment variables *MF_PROVISION_CERTS_CA* & *MF_PROVISION_CERTS_CA_KEY* which seems not used by the *main.go*.

File: docker/addons/provision/docker-compose.yml [hunk 2]
```diff
@@ -25,11 +25,9 @@ services:
       MF_PROVISION_LOG_LEVEL: ${MF_PROVISION_LOG_LEVEL}
       MF_PROVISION_HTTP_PORT: ${MF_PROVISION_HTTP_PORT}
       MF_PROVISION_CONFIG_FILE: ${MF_PROVISION_CONFIG_FILE}
-      MF_PROVISION_LOG_LEVEL: ${MF_PROVISION_LOG_LEVEL}
```
Comment (drasko) [inline docker/addons/provision/docker-compose.yml:28]: Why are these removed? Not used?

**Review by drasko — CHANGES_REQUESTED** [review]

File: docker/addons/provision/docker-compose.yml (hunk 2 above)
Comment (jonathandreyer) [reply docker/addons/provision/docker-compose.yml:28]: @drasko It is declared twice (three line above)

File: users/postgres/init.go [hunk 3]
```diff
@@ -96,6 +96,13 @@ func migrateDB(db *sqlx.DB) error {
 				)`,
 				},
 			},
+			{
+				Id: "users_5",
+				Up: []string{
```
Comment (nmarcetic) [inline users/postgres/init.go]: Maybe you should move this to dedicated PR, so we keep the clean history (this one is important).
Let's follow an established rule for not refactoring the code or make changes, not related to a specific feature/fix that you are working on.

**Review by nmarcetic — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM
